src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/ClaimFeeRequiredScenarioTest.java

```diff
@@ -14,9 +12,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
-@Testcontainers
-public class ClaimFeeRequiredScenarioTest extends BaseIntegrationTest {
+public class ClaimFeeRequiredScenarioTest extends DashboardBaseIntegrationTest {
 
     public static final String SCENARIO_CLAIM_FEE_REQUIRED = "Scenario.AAA6.ClaimIssue.ClaimFee.Required";
     private static final String DASHBOARD_CREATE_SCENARIO_URL
```

Comment:
Should this file have the change as well?

Reply:
added missed tests